cpp/core/jni/JniWrapper.cc

```diff
@@ -580,17 +583,28 @@ Java_org_apache_gluten_vectorized_NativeColumnarToRowJniWrapper_nativeColumnarTo
     JNIEnv* env,
     jobject wrapper,
     jlong batchHandle,
-    jlong c2rHandle) {
+    jlong c2rHandle,
+    jlong rowId) {
```

Comment:
Can we pass a range [startRowId, endRowId) to naitve ? I think it should be more clear to use the config at java side.